corehq/apps/userreports/tasks.py

```diff
@@ -170,7 +170,7 @@ def _run_report(backend_to_use):
             json_response["total_row"] = total_row
         return json_response
 
-    spec = get_document_or_not_found(ReportConfiguration, domain, report_config_id)
```

Comment:
Isn't there any code that is expecting an exception if config_id is not found, does this change handle that?